app/components/LoginForm/LoginPage.js

```diff
@@ -1,15 +1,71 @@
-import React from 'react';
-import styles from './Login.css';
+// @flow
+
+import React, { Component } from 'react';
 import { Container } from '../Layout';
-import LoginForm from './LoginForm';
+import { LoginForm, RegisterForm, ForgotPasswordForm } from '../LoginForm';
+import styles from './Login.css';
+import NavigationTab, { NavigationLink } from 'app/components/NavigationTab';
+import { Link } from 'react-router';
+
+type State = {
+  mode: 'login' | 'register' | 'forgotPassword'
+};
+
+class LoginPage extends Component<null, State> {
+  state: State = {
+    mode: 'login'
+  };
+
+  toggleRegisterUser = (e: Event) => {
+    this.setState({ mode: 'register' });
+  };
+
+  toggleForgotPassword = (e: Event) => {
+    this.setState({ mode: 'forgotPassword' });
+  };
 
-const LoginPage = props => (
-  <Container>
-    <div className={styles.root}>
-      <h2>Logg inn</h2>
-      <LoginForm />
-    </div>
-  </Container>
-);
+  toggleBack = (e: Event) => {
+    this.setState({ mode: 'login' });
+  };
 
+  render() {
+    const isLogin = this.state.mode === 'login';
+    let title, form;
+    switch (this.state.mode) {
+      case 'login':
+        title = 'Logg inn';
+        form = <LoginForm />;
+        break;
+      case 'register':
+        title = 'Registrer bruker';
+        form = <RegisterForm />;
+        break;
+      case 'forgotPassword':
+        title = 'Glemt passord';
+        form = <ForgotPasswordForm />;
+        break;
+    }
+    return (
+      <Container className={styles.window}>
```

Comment:
The loginpage should certainly have a white background, and it doesn't seem to act too nicely on smaller screens. I'm not sure how `Content` would work here, it might make the component 100% width which you might not want, but you should test it out. Gotta rebase with master to get access to it. If `Content` doesn't make sense then you could compose some style or manually style it, just make sure it gets a nice white background and works well on all screen sizes.